documents/src/pages/elements/slider.md

```diff
@@ -271,41 +271,46 @@ const slider = document.getElementById("movement-speed");
 
 container.addEventListener("resize", (event) => {
   const containerWidth = event.detail.width;
-  containerWidth < 200 ? slider.setAttribute("very-small", "") : slider.removeAttribute("very-small");
-  containerWidth < 400 ? slider.setAttribute("small", "") : slider.removeAttribute("small");
+  containerWidth <= 200 ? slider.setAttribute("small", "") : slider.removeAttribute("small");
+  containerWidth <= 400 ? slider.setAttribute("medium", "") : slider.removeAttribute("medium");
 });
 
-slider.addEventListener("value-changed", () => {
-  container.style.setProperty('--speed', `${4-((slider.value/100)*3)}s`);
+function onChangeRadio(event) {
+  container.style.width = event.target.value;
+}
+
+document.querySelectorAll("ef-radio-button").forEach((element) => {
+  element.addEventListener('checked-changed', onChangeRadio);
 });
 ```
 ```css
-@keyframes resize {
-  from {
-    width: 600px;
-  }
-  to {
-    width: 150px;
-  }
+#radio-group {
+  margin: 10px;
 }
 #container {
   margin: 20px;
-  animation: resize ease-in-out var(--speed, 4s) alternate infinite;
+  width: 600px;
 }
 ef-slider {
   padding: 0 5px;
 }
 ef-slider {
   padding: 0 5px;
 }
-ef-slider[small] > ef-slider-marker[important-low] {
+ef-slider[medium] > ef-slider-marker[important-low] {
   display: none;
 }
-ef-slider[very-small] > ef-slider-marker[important-med] {
+ef-slider[small] > ef-slider-marker[important-med] {
   display: none;
 }
 ```
 ```html
+<div id="radio-group">
+  <p>Select width of slider</p>
+  <ef-radio-button name="size" value="200px">Small</ef-radio-button>
+  <ef-radio-button name="size" value="400px">Medium</ef-radio-button>
+  <ef-radio-button name="size" value="600px" checked>Large</ef-radio-button>
+</div>
```

Comment:
please check comments on https://github.com/Refinitiv/refinitiv-ui/pull/1069

Reply:
Updated. Thank you P'Ole.